### DISCUSSION TITLE
tutorials/lift-off-part2/09-errors-when-queries-go-sideways

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#13] [course feedback]
# tutorials/lift-off-part2/09-errors-when-queries-go-sideways

https://www.apollographql.com/tutorials/lift-off-part2/09-errors-when-queries-go-sideways

**Comment by Deepak22448** [comment]
I liked the Code Challenge! from this class because it makes me more confident.

**Comment by EslemOuederni** [comment]
Great course once again !

**Comment by CMH7** [comment]
I really like and love this course much much much more easier

**Comment by avioXD** [comment]
Really great course! As a beginner I understood all methods clearly.

**Comment by anshumanfs** [comment]
Awesome Course and examples💕❤️❤️

**Comment by catreunion** [comment]
Dear authors, thank you so much for giving me the key to happiness. 🙏

**Comment by klvnmarshall** [comment]
Awesome course

**Comment by neogeek13** [comment]
Consider rewording this question:
Within a server response, what keys can you expect to find in the JSON object?

The answers being considered correct are data + errors. But the errors key will not be included when there are none. While it makes sense in the context of this lesson, rewording this question as follows will add clarity:
Within a server response that encounters problems, what key(s) can you expect to find in the JSON object?

OR 

Within the server response in the context of this module/lesson, what keys can you expect to find in the JSON object?

**Comment by tvadiep** [comment]
Love it, it make me more confident!

**Comment by nwazuo** [comment]
Okay. 'The key to happinness' option in the short quiz is really funny😅

**Comment by Yasir5247** [comment]
I wish it was key to happiness that it returns. :P

**Reply by amaimasque** [reply]
Same

**Comment by pgn666** [comment]
english is not my first language, but isnt "things can go south" not fully political correct langunage?

**Reply by pgn666** [reply]
I did some reaserch. The phrase seems to be OK. Maybe I'm not by thinkings its worng...

**Reply by lizhennessy** [reply]
Thanks for pointing this out, @pgn666 ! That phrase has always made me think of a chart with numbers going down (presumably to zero) but I think it's important for us to consider whether or not this language is accessible for all of our learners - particularly those who speak English as a second language! We'll take a look through the course content and make sure that all of these idiomatic phrases are clear and free of negative associations.

**Comment by talwar** [comment]
the course is really well designed. A lot of thought and hard work has gone into it.

**Comment by LingVuDev** [comment]
I found the key to happiness ❤️, even though its wrong :D
